test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/rest/RestClient.java

```diff
@@ -43,11 +60,26 @@ public Response postCommand(final String url, final String contentType, final St
     public Response postCommand(final String url, final String contentType, final String requestPayload, final MultivaluedMap<String, Object> headers) {
         final Entity<String> entity = entity(requestPayload, MediaType.valueOf(contentType));
 
-        return clientBuilder().build()
+
+        if (LOGGER.isInfoEnabled()) {
+            LOGGER.info("Making POST request to '{}' with Content Type '{}'", url, contentType);
+            LOGGER.info("Request payload: '{}'", requestPayload);
+            LOGGER.info("Headers: {}", headers);
+        }
+
+
+        final Response response = clientBuilder().build()
                 .target(url)
                 .request()
                 .headers(headers)
                 .post(entity);
+
+        if (LOGGER.isInfoEnabled()) {
```

Comment:
A good point - however, by putting the logging into a private method, you loose the correct line number in the logs, which can be kind of useful.